GliaWidgets/Theme/Theme+Call.swift

```diff
@@ -78,7 +78,7 @@ extension Theme {
             secondTextFontColor: color.baseLight,
             accessibility: .init(
                 firstTextHint: Accessibility.Connect.Queue.FirstText.hint,
-                secondTextHint: Accessibility.Connect.Queue.SecondText.hint,
+                secondTextHint: nil,
```

Comment:
At this point to we need to have `secondTextHint ` at all?

Reply:
It's needed in `connected` case.